ember_debug/models/profile-manager.js

```diff
@@ -80,6 +162,53 @@ export default class ProfileManager {
     }
   }
 
+  teardown() {
+    this.stylesheet.remove();
+    // remove all the active highlighted components
+    for (const key in this.highlightComponents) {
+      const elementId = getNodeId(key);
```

Comment:
seems like we should have ability to cancelTimeouts here, to avoid some side-effects

Looks like we could have array-based higlight manager:

```js
higlights = [];
removeHiglight(el: { timeout: number, el: DOMNode }) {
   higlights = higlights.filter(item => item !== el);
  cancelTimout(el.timeout);
  el.node.remove();
}
addHiglight(el) {
   this.higlights.push(el);
   el.timeout = setTimout(...);
  // ...
}
```

Reply:
updated.